packages/flutter_tools/lib/src/web/compile.dart

```diff
@@ -28,10 +28,13 @@ Future<void> buildWeb(
   bool nativeNullAssertions,
   String? baseHref,
   String? dart2jsOptimization,
+  [String? outputDirectoryPath,]
```

Comment:
Use a named parameter, because otherwise things will get awkward the next time we add an optional parameter

```suggestion
  {String? outputDirectoryPath,}
```